### PR TITLE
get group by container

### LINKED ISSUE #202: Get Group By Container ID

### PR DESCRIPTION
**Pull request by hitohata**
## Overviews of implementation

Updated the get group method to implement the `get group by container` access pattern.

## Review points

Please check the comment in the line on GitHub.

### CODE REVIEW

File: backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts [hunk 1]
```diff
@@ -193,26 +199,81 @@ export class NishikiDynamoDBClient {
...
+
+				throw new NishikiTableClientError(
+					"There are more than one groups in the container.",
+					containerIds,
+				);
+			}
+
+			const result = unmarshall(response.Items[0]);
+
+			return this.getGroup({ groupId: result.PK });
```
Comment (hitohata) [inline backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts:269]: To fulfill the return type, this method calls `getGroup` method again. This looks redundant.
To fix this, we need to change the table definition, adding the `GroupName` attribute.

Is it necessary? What do you think?

Comment (taniguchiiqqq) [reply backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts:269]: @hitohata 
I understood your point. Sorry, that's my bad. I think we should change the table definition like updating the ProjectionType of the GroupContainerRelationship to All or INCLUDE.
But for now, how about keeping going with your implementation? Then, I'll make issues in the document repo and backend repo.

**Review by taniguchiiqqq — APPROVED** [review]
Thank you for implementation!

File: backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts (hunk 1 above)
Comment (hitohata) [reply backend/main/src/Shared/Adapters/DB/NishikiTableClient.ts:269]: @taniguchiiqqq 
OK, thank you. I'm looking forward to you opening new issues.

**Pull request merged**